src/elements/core/mixins/form-associated-mixin.ts

```diff
@@ -36,17 +36,17 @@ export declare abstract class SbbFormAssociatedMixinType {
  * The FormAssociatedMixin enables native form support for custom controls.
  */
 // eslint-disable-next-line @typescript-eslint/naming-convention
-export const SbbFormAssociatedMixin = <T extends Constructor<LitElement>>(
+export const SbbFormAssociatedMixin = <T extends AbstractConstructor<LitElement>, V = string>(
```

Comment:
Nitpick; In the named slot mixin, the order of the generic type and the LitElement constructor type is the other way around.

Reply:
Since it is "optional", I had to put it last